tests/testAutopas/tests/autopasInterface/DifferentParticlesTest.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file DifferentParticlesTest.cpp
+ * @author seckler
+ * @date 20.02.2020
+ */
+
+#include "DifferentParticlesTest.h"
+
+#include "autopas/AutoPas.h"
+#include "testingHelpers/commonTypedefs.h"
+
+/**
+ * A particle class without an actual constructor (only copy, etc.).
+ */
+class NonConstructibleParticle : public Particle {
+ public:
+  /**
+   * Default constructor.
+   */
+  NonConstructibleParticle() = default;
+};
+
+/**
+ * Tests if AutoPas still compiles with a Particle that implements the normal interface, BUT no constructor.
+ */
+TEST_F(DifferentParticlesTest, testNonConstructibleParticle) {
+  autopas::AutoPas<NonConstructibleParticle, autopas::FullParticleCell<NonConstructibleParticle>> autoPas;
+  autoPas.setBoxMin({0., 0., 0.});
+  autoPas.setBoxMax({10., 10., 10.});
+  autoPas.setCutoff(1.);
```

Comment:
there seems to be no need to set those. Maybe only boxMax. I actually don't know if something reacts allergic to domains without any volume...